### PR TITLE
Redux

### PR DESCRIPTION
**Pull request by DiscoElevator**

### CODE REVIEW

File: app/src/pages/outside/loginPage/loginPage.scss [hunk 1]
```diff
@@ -40,19 +40,19 @@
   left: 0;
   width: 50%;
   height: 100%;
-  background-image: url(./img/login-page-bgc.svg);
+  background-image: url(img/login-page-bgc.svg);
```
Comment (ArtsiomYeliseyenka) [inline app/src/pages/outside/loginPage/loginPage.scss]: Now we will use the absolute paths for images?

File: app/src/pages/outside/loginPage/loginPage.scss [hunk 2]
```diff
@@ -40,19 +40,19 @@
...
   background-color: $COLOR--light-blue;
   background-repeat: no-repeat;
   background-size: 100%;
 
   @media (max-width: $SCREEN_SM_MAX) {
     width: 100%;
-    background-image: url(./img/login-page-tablet-bgc.svg);
+    background-image: url(img/login-page-tablet-bgc.svg);
```
Comment (lexecon) [inline app/src/pages/outside/loginPage/loginPage.scss]: use the relative paths for images

File: app/src/pages/outside/loginPage/loginPage.scss [hunk 3]
```diff
@@ -61,7 +61,7 @@
   left: 60px;
   width: 140px;
   height: 23px;
-  background-image: url(./img/rp-logo.svg);
+  background-image: url(img/rp-logo.svg);
```
Comment (lexecon) [inline app/src/pages/outside/loginPage/loginPage.scss]: use the relative paths for images

File: app/src/pages/outside/loginPage/socialSection/newsBlock/newsBlock.scss [hunk 4]
```diff
@@ -23,7 +23,7 @@
   height: 20px;
   margin-bottom: -5px;
   margin-right: 5px;
-  background-image: url(./img/twitter-gray.svg);
+  background-image: url(img/twitter-gray.svg);
```
Comment (lexecon) [inline app/src/pages/outside/loginPage/socialSection/newsBlock/newsBlock.scss]: use the relative paths for images

File: app/src/pages/outside/loginPage/socialSection/socialsBlock/socialsBlock.scss [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+  margin-right: 33px;
+  background-repeat: no-repeat;
+  background-position: center;
+
+  @media (max-width: $SCREEN_SM_MAX) {
+    margin-right: 60px;
+  }
+
+  &.gh-icon {
+    background-image: url(img/socialGh.svg);
```
Comment (lexecon) [inline app/src/pages/outside/loginPage/socialSection/socialsBlock/socialsBlock.scss]: use the relative paths for images

File: app/src/pages/outside/registrationPage/registrationFailBlock/registrationFailBlock.scss [hunk 6]
```diff
@@ -3,7 +3,7 @@
   height: 100%;
 
   @media (max-width: $SCREEN_SM_MAX) {
-    background-image: url(./img/registration-fail-form-tablet.svg);
+    background-image: url(img/registration-fail-form-tablet.svg);
```
Comment (lexecon) [inline app/src/pages/outside/registrationPage/registrationFailBlock/registrationFailBlock.scss]: use the relative paths for images

File: app/src/pages/outside/registrationPage/registrationPage.scss [hunk 7]
```diff
@@ -40,33 +40,33 @@
   left: 0;
   width: 50%;
   height: 100%;
-  background-image: url(./img/registration-page-bgc.svg);
+  background-image: url(img/registration-page-bgc.svg);
```
Comment (lexecon) [inline app/src/pages/outside/registrationPage/registrationPage.scss]: use the relative paths for images

**Pull request merged**